MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
```

Comment:
@1ec5 this does not account for `ref.hasPrefix("US ")` though. `contains` allows us to add more cases to this array.